### DISCUSSION TITLE
Blueprint Big Book of Data Analysis - Workflow

### DISCUSSION THREAD

**Discussion by giscus** [context-dependent/bp-big-book#5] [General]
# Blueprint Big Book of Data Analysis - Workflow

https://context-dependent.github.io/bp-big-book/wf.html

**Comment by mpalamar-bp** [comment]
Can we say more about "why"? I think a more detailed (but still ultimately concise) rationale about the value of using a version/project control system like Github will help it "click" more for people in how this will help lead to better practice

**Reply by context-dependent** [reply]
See issue #7
